silverstripe/framework/CVE-2020-26138.yaml

```diff
@@ -0,0 +1,8 @@
+title:     'CVE-2020-26138 FormField: with square brackets in field name skips validation'
+link:      https://www.silverstripe.org/download/security-releases/cve-2020-26138
+cve:       CVE-2020-26138
+branches:
+    4.0.x:
+        time:     2021-06-07 22:31:00
+        versions: ['>=4.0.0', '<4.8.0']
```

Comment:
The linked CVE says affected versions are "silverstripe/framework: ^3.0.0, ^4.0.0" and 4.7.4 was also a fixed version, so I think you need to add additional branches here?